lib/appsync-events-client.ts

```diff
@@ -2,21 +2,67 @@
  * AWS AppSync Events WebSocket subprotocol identifier
  * @internal
  */
-const AWS_APPSYNC_EVENTS_SUBPROTOCOL = 'aws-appsync-event-ws'
+const AWS_APPSYNC_EVENTS_SUBPROTOCOL = "aws-appsync-event-ws";
+
+/**
+ * Configuration options for AppSyncEventsClient
+ * @public
+ */
+/**
+ * Amplify auth configuration for AppSync
+ * @public
+ */
+type JsonPrimitive = null | string | number | boolean;
+
+interface JwtPayloadStandardFields {
```

Comment:
in general, I dont think we should make this an amplify thing. let's be more generic, we also dont need to implement any JWT Stuff, we just want users to supply a token.